### PR TITLE
Issue #7604: Update doc NonEmptyAtclauseDescriptionCheck

### PR DESCRIPTION
**Pull request by ImmortalRabbit**
add default example

### CODE REVIEW

**Comment by ImmortalRabbit** [comment]
I have an issue with this pull request because examples are not executable because `// Violation` commentary is counted as Javadocs for tag like `@param`, they supposed to be an empty in order to fail but I wanted to make a note where violation occurs. Is there any suggestion, how I can make an example which will be executable and with a violation note.

**Comment by timurt** [comment]
@ImmortalRabbit 

> I have an issue with this pull request because examples are not executable because // Violation commentary is counted as Javadocs for tag like @param, they supposed to be an empty in order to fail but I wanted to make a note where violation occurs. Is there any suggestion, how I can make an example which will be executable and with a violation note.

I checked `NonEmptyAtclauseDescriptionCheckTest` and violations are described inside unit tests, not in tested class, probably you can do the same thing

But I am not sure that writing unit tests is part of this issue

**Comment by pbludov** [comment]
> please add examples for `violateExecutionOnNonTightHtml = true` parameter

@timurt please no. This will confuse users as it is not related to the Check itself.

File: src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java [hunk 1]
```diff
@@ -57,6 +57,22 @@
...
+ * &#47;**
+ * *
+ * * &#64;param a Some javadoc // OK
+ * * &#64;param b // Violation - At-clause should have a non-empty description.
+ * * &#64;deprecated // Violation - At-clause should have a non-empty description.
+ * * &#64;throws Exception // Violation - At-clause should have a non-empty description.
+ * *&#47;
+ * public int method(String a, int b) throws Exception
+ * {
+ * return 1;
```
Comment (romani) [inline src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/NonEmptyAtclauseDescriptionCheck.java:72]: please make proper indentation for code in javadoc.

please run javadoc jdk tool over this file and share result (as image) in PR description. javadoc should render properly and code is easy to read there.

**Comment by romani** [comment]
@ImmortalRabbit , ping, please finish PR.

**Review by romani — APPROVED** [review]

**Pull request merged**